molecule/active-ubuntu2004/Dockerfile.j2

```diff
@@ -6,7 +6,7 @@ FROM {{ item.registry.url }}/{{ item.image }}
 FROM {{ item.image }}
 {% endif %}
 
-RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get install -y python sudo bash ca-certificates && apt-get clean; \
+RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get install -y python sudo bash ca-certificates gnupg && apt-get clean; \
```

Comment:
Should this be moved to a `prepare.yml` playbook?

Reply:
I can try to move the gnupg. In my tries, the success of installing something via the ``prepare.yml`` playbook was Zero. The complaint was always ``omero-web-active-ubuntu2004]: FAILED! => {"changed": false, "msg": "No package matching 'xxx' is available"} . Replace xxx with `curl`, gnupg etc as appropriate.

Reply:
This is fixed as far as I can with https://github.com/ome/ansible-role-omero-web/pull/45/commits/65e2b70461f7740539ea06e6586aa4df17a8ff85

The reasoning is:

1. Tried to remove the Dockerfile completely and put the packages installs into the [prepare.yml](https://github.com/ome/ansible-role-omero-web/pull/45/commits/65e2b70461f7740539ea06e6586aa4df17a8ff85#diff-b5f60a491d9c8f17eb774de1a16d0d8eec5bc6f14e25355e910212fb5b11a4c9). This failed with subsequent report that pkg such-and-such (in the example below it is ``curl``, the first in the list) cannot be installed:

https://github.com/ome/ansible-role-omero-web/actions/runs/5254245234/jobs/9492614696

``no package matching curl is available``

2. Returned the Dockerfile back, but deleted all the package installs from it. This was working, except for sudo, which failed with


https://github.com/ome/ansible-role-omero-web/actions/runs/5254389653/jobs/9492935545:
```
TASK [Install packages] ********************************************************
  fatal: [omero-web-active-ubuntu2004]: FAILED! => {"changed": false, "module_stderr": "/bin/sh: 1: sudo: not found\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 127}